src/main/java/org/gridsuite/shortcircuit/server/repositories/FaultResultRepository.java

```diff
@@ -22,7 +25,13 @@
  */
 @Repository
 public interface FaultResultRepository extends JpaRepository<FaultResultEntity, UUID> {
+    Page<FaultResultEntity> findAllByResult(ShortCircuitAnalysisResultEntity result, Pageable pageable);
```

Comment:
maybe name the methods that take a pageable "findPagedXXX" instead of findAll

Reply:
done